src/etc/inc/captiveportal.inc

```diff
@@ -347,14 +347,14 @@ EOD;
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-error.html");
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-logout.html");
 
-		captiveportal_radius_stop_all();
+		captiveportal_disconnect_all(10); // NAS-Request
 
 		captiveportal_filterdns_configure();
 
 		/* send Accounting-Off to server */
-		if (!platform_booting()) {
+		/*if (!platform_booting()) {
 			captiveportal_send_server_accounting(true);
-		}
+		}*/
```

Comment:
That's the reason I pushed the patch as an RFC, I wasn't sure that I understood why the function was even called. I planned to remove it in the final patch, after giving another look at the function, unless someone pointed me to a reason to keep it.